artemis/src/main/java/tech/pegasys/artemis/cli/deposit/EncryptedKeystoreWriter.java

```diff
@@ -56,18 +56,13 @@ public void writeKeys(final BLSKeyPair validatorKey, final BLSKeyPair withdrawal
     final KeyStoreData withdrawalKeyStoreData =
         generateKeystoreData(withdrawalKey, withdrawalKeyPassword);
 
-    saveKeyStore(
-        keystoreDirectory.resolve("validator_" + validatorKey.getPublicKey().toString() + ".json"),
-        validatorKeyStoreData);
-    saveKeyStore(
-        keystoreDirectory.resolve(
-            "withdrawal_" + withdrawalKey.getPublicKey().toString() + ".json"),
-        withdrawalKeyStoreData);
+    saveKeyStore(keystoreDirectory.resolve("validator.json"), validatorKeyStoreData);
+    saveKeyStore(keystoreDirectory.resolve("withdrawal.json"), withdrawalKeyStoreData);
```

Comment:
Given that not using the full public key opens up the possibility of overwriting an existing key with a different value, we should add a check in `saveKeyStore` so that it fails if the file already exists.  The chance of such a conflict is extremely low but given it results in a loss of funds we want to completely eliminate any risk of it.

Given how rare the overlap is, just aborting before we send the deposit would be fine.